compiler/llvm/llvm-global-to-wide/RUN_CONFIG

```diff
@@ -12,4 +12,4 @@ ln -s ../../include/llvmVer.h llvmVer.h
 mkdir -p build
 cd build
 export CMAKE_PREFIX_PATH=$CHPL_HOME/third-party/llvm/install/linux64-gnu/
-cmake .. -DLLVM_ROOT=$CHPL_HOME/third-party/llvm/install/linux64-gnu/ -DLLVM_SRC=$CHPL_HOME/third-party/llvm/llvm
+cmake .. -DLLVM_ROOT=$CHPL_HOME/third-party/llvm/install/linux64-gnu/ -DLLVM_SRC=$CHPL_HOME/third-party/llvm/llvm -DCMAKE_BUILD_TYPE=Debug
```

Comment:
This looks like we are always forcing debug mode.  Is that right?

Reply:
For this script, yes, but AFAIK this script is only ever used for debugging & testing the llvm-global-to-wide passes.